modules/core/src/main/java/org/apache/ignite/internal/processors/cache/GridCacheUtils.java

```diff
@@ -1578,6 +1581,17 @@ public static TransactionConfiguration transactionConfiguration(@Nullable final
      */
     public static void validateCacheName(String name) throws IllegalArgumentException {
         A.ensure(name != null && !name.isEmpty(), "Cache name must not be null or empty.");
+        A.ensure(name.length() <= MAX_FILE_NAME_LENGTH, "Length of cache name can not exceed "+ MAX_FILE_NAME_LENGTH +".");
+    }
+
+    /**
+     *
+     * @param name
+     * @throws IllegalArgumentException In case group name is not valid
+     */
+    public static void validateCacheGroupName(String name) throws IllegalArgumentException {
+        if (name != null)
+            A.ensure(name.length() <= MAX_FILE_NAME_LENGTH, "Length of cache group name can not exceed "+ MAX_FILE_NAME_LENGTH +".");
```

Comment:
Code style issue - there should be whitespace between `"` and `+`

Reply:
done!